run.py

```diff
@@ -32,7 +32,7 @@ def str_to_bool(s):
     parser_search.set_defaults(run=manager.search)
     parser_search.add_argument("--repo", required=True, type=str, help="the image repo (i.e., osirrc2019/anserini)")
     parser_search.add_argument("--tag", default="latest", type=str, help="the image tag (i.e., latest)")
-    parser_search.add_argument("--save_id", default="save", type=str, help="determines the ID of saved image (to search from)")
+    parser_search.add_argument("--load_from_tag", default="save", type=str, help="used to determine the snapshotted image to search from")
```

Comment:
I would specify that "it's used to determine the tag of the snapshotted image to search from".